@market/hooks/useAskHelper.ts

```diff
@@ -31,7 +31,7 @@ export const useAskHelper = ({ ask }: AskHelperProps) => {
   const usdAskAmount = useMemo(
     () =>
       ask?.amount?.usd?.value
-        ? numberFormatter(roundTwoDecimals(ask?.amount?.usd?.value))
+        ? numberFormatterUSDC(roundTwoDecimals(ask?.amount?.usd?.value))
```

Comment:
here we don't need to check with `?`, since `ask?.amount?.usd?.value` is checked above